backend/src/main/java/com/woowacourse/friendogly/footprint/service/FootprintQueryService.java

```diff
@@ -58,6 +58,7 @@ public List<FindNearFootprintResponse> findNear(Long memberId, FindNearFootprint
 
         return recentFootprints.stream()
                 .filter(footprint -> footprint.isNear(currentLocation))
+                .filter(footprint -> !footprint.isDeleted())
                 .map(footprint -> new FindNearFootprintResponse(footprint, footprint.isCreatedBy(memberId)))
```

Comment:
```suggestion
        return recentFootprints.stream()
                .filter(footprint -> !footprint.isDeleted() && footprint.isNear(currentLocation))
                .map(footprint -> new FindNearFootprintResponse(footprint, footprint.isCreatedBy(memberId)))
```

혹시라도 범위 내 조회가 부활한다면 이렇게 수정하면 좋을 것 같아요.

1. filter를 중복으로 쓰면 불필요한 객체 생성이 일어나서 성능에 좋지 못해요. [참고 링크](https://www.baeldung.com/java-streams-multiple-filters-vs-condition)
2. 나중에 데이터가 쌓이면 논리 삭제 되어 있는 발자국이, 그렇지 않은 발자국보다 많아질 것 같아요. `&&`는 도도도 잘 아시겠지만 앞의 조건이 false이면 뒤 조건을 검사하지 않기 때문에, 이렇게 바꾸면 불필요한 `isNear` 검사가 줄어들 것 같네요!